### DISCUSSION TITLE
Azure image for upgrade from 2.5.3 to 2.7.0

### DISCUSSION THREAD

**Discussion by williehao** [ant-media/Ant-Media-Server#5835] [FAQs] [answered]
I attempted to upgrade from version 2.5.3 EE to 2.7.0 EE via GitHub , but unfortunately, I encountered an issue. Could you please guide me on where to find the correct ZIP file?
PS: Environment: AZURE VM with AMS (License from AZURE Marketplace)

References 
https://github.com/ant-media/Ant-Media-Server/releases

[Upgrading Ant Media Server | Ant Media Documentation](https://antmedia.io/docs/guides/installing-on-linux/upgrading-ant-media-server/)

**Comment by Mohit-3196** [comment, marked as answer]
Hi @williehao,
How are you?

I found you that you have created a support ticket for this issue as well and I just replied to you there.
Let me forward the message here as well.

Yes, you need to have the installation zip file in order to make the upgrade. The zip that you got from https://github.com/ant-media/Ant-Media-Server/releases is the community edition zip file. So I have shared the marketplace upgrade zip file with you on the mail. 

If you have already upgraded and see some kind of issues, please roll back to v2.5.3, and for that, you can follow these steps https://antmedia.io/docs/guides/installing-on-linux/upgrading-ant-media-server/#how-do-i-restore-ant-media-server-if-needed

Then you can upgrade your server to v2.7.0
You can follow these steps to upgrade https://antmedia.io/docs/guides/installing-on-linux/upgrading-ant-media-server/

Thank you.
